build.gradle

```diff
@@ -1,6 +1,54 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.palantir.docker' version '0.21.0'
+    id 'com.palantir.docker-compose' version '0.21.0'
```

Comment:
✏️ Is this dicker stuff related to your PR?

Reply:
Not this one, but it'l be needed as soon as we implement docker. I'll wait on it